### ISSUE TITLE
the aspect ratio of cairo_circuit_map cannot be changed

### ISSUE THREAD

**Issue by yuanduopeng** [time4tea/gopro-dashboard-overlay#150]
My layout xml is:
```
    <composite x="30" y="50" name="map">
        <component type="cairo_circuit_map" size="256" rotate="-90" fill="255,255,255" 
            loc-fill="255,255,255"
            outline="0,0,0" loc-outline="0,0,0" />
    </composite>
```
But the ratio of cairo circuit map is so weird like below:
![image](https://github.com/time4tea/gopro-dashboard-overlay/assets/65839388/fbc2df71-48f7-4262-807f-5a8a2297665d)   
The correct map aspect ratio is like below:
![image](https://github.com/time4tea/gopro-dashboard-overlay/assets/65839388/2d7da422-f8b8-4ecc-a263-44e02a1653b2)

**Comment by time4tea** [comment]
Hmm. yeah i see what you mean.

**Comment by time4tea** [comment]
This should be fixed in v0.105.0 - should be on PyPI by now. Thanks for raising!

**Comment by yuanduopeng** [comment]
Thanks a lot!!! It was fixed successfully. 
![image](https://github.com/time4tea/gopro-dashboard-overlay/assets/65839388/651561c1-75e6-4fd6-b461-0d9528c2e71b)
